src/main/java/org/isf/supplier/dto/SupplierDTO.java

```diff
@@ -32,25 +32,25 @@ public class SupplierDTO {
 	private Integer supId;
 	
 	@NotNull(message="supplier's name is required")
-	@Schema(description="The supplier's name", example = "Cogefar")
+	@Schema(description="The supplier's name", example = "Cogefar", maxLength = 100)
 	private String supName;
 	
-	@Schema(description="The supplier's address", example = "25 Rue Ministre, Dschang")
+	@Schema(description="The supplier's address", example = "25 Rue Ministre, Dschang", maxLength = 150)
 	private String supAddress;
 	
-	@Schema(description="The supplier's tax code", example = "5221")
+	@Schema(description="The supplier's tax code", example = "5221", maxLength = 50)
 	private String supTaxcode;
 	
-	@Schema(description="The supplier's phone", example = "+237654120145")
+	@Schema(description="The supplier's phone", example = "+237654120145", maxLength = 20)
 	private String supPhone;
 	
-	@Schema(description="The supplier's fax number", example = "+237654120145")
+	@Schema(description="The supplier's fax number", example = "+237654120145", maxLength = 20)
```

Comment:
Shouldn't this be the same length as phone numbers?

Reply:
I think we should make all the same, starting by the DB...

Reply:
![supplier](https://github.com/informatici/openhospital-api/assets/101590821/475fef52-8aec-466a-8210-1d1e85511f78)
I use the lenght specify in the database

Reply:
> I think we should make all the same, starting by the DB... not in this PR.
> 
> 20 chars sound reasonable and enough, wdyt?
I agree with you